tests/openvino/native/common.py

```diff
@@ -11,15 +11,20 @@
  limitations under the License.
 """
 import os
+import json
```

Comment:
Imports should be grouped in the following order:
- Standard library imports.
- Related third-party imports.
- Local application/library-specific imports.

You should put a blank line between each group of imports.

https://github.com/openvinotoolkit/nncf/blob/develop/docs/styleguide/PyGuide.md#22-imports